package.json

```diff
@@ -29,50 +32,46 @@
     "prepublish": "yarn run build"
   },
   "dependencies": {
-    "babel-plugin-react-flow-props-to-prop-types": "0.15.0",
```

Comment:
we don't need this any more?

Reply:
This is probably the most contentious part of the PR. This is used to generate prop types from flow. However, it is just one more thing to maintain and makes it a little hard to move forward quickly. My thought is to drop it given that we publish flow types and there is public typescript types also. 

What are your thoughts?

Reply:
Update: we were not generating prop types anyway. Given that nobody has complained I think we just leave it as is